verifier/src/main/kotlin/tech/figure/classification/asset/verifier/client/VerifierClient.kt

```diff
@@ -86,7 +87,7 @@ class VerifierClient(private val config: VerifierClientConfig) {
         return startVerifying(startingBlockHeight)
     }
 
-    private suspend fun verifyLoop(startingBlockHeight: Long?) {
+    private suspend fun verifyLoop(startingBlockHeight: Long?, restartCount: Long = 0) {
```

Comment:
Including this count parameter will ensure that manual restarts will reset the restart counter to 0 automatically.  This removes the need for a globally-tracked restart counter in the client itself.